index.js

```diff
@@ -30,8 +30,15 @@ function partitionBundle(b, opts) {
   // require the modules from the map
   forOwn(opts.map, function(modules, file) {
     modules.forEach(function(mod, i) {
-      var id = bresolve.sync(mod, rOpts);
-      shortIDLabels[id] = mod;
+      if (typeof mod === 'string') {
+        mod = {
+          require: mod,
+          expose: mod
+        }
```

Comment:
Can you add a semicolon here?

See https://travis-ci.org/arian/partition-bundle/jobs/103460093#L326